src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/index.tsx

```diff
@@ -304,12 +299,24 @@ class DebugImageDetails extends AsyncComponent<Props, State> {
           </Content>
         </Body>
         <Footer>
-          <Button
-            href="https://docs.sentry.io/platforms/native/data-management/debug-files/"
-            external
-          >
-            {t('Read the docs')}
-          </Button>
+          <StyledButtonBar>
+            <Button
+              href="https://docs.sentry.io/platforms/native/data-management/debug-files/"
+              external
+            >
+              {t('Read the docs')}
+            </Button>
+            {debugFilesSettingsLink && (
+              <Tooltip
+                title={t(
+                  'Search for this debug file in all images for the %s project',
+                  projectId
+                )}
+              >
+                <Button to={debugFilesSettingsLink}>{t('Search in Settings')}</Button>
+              </Tooltip>
```

Comment:
We can use `title` prop on `Button` component
```suggestion
                <Button to={debugFilesSettingsLink} title={t(
                  'Search for this debug file in all images for the %s project',
                  projectId
                )}>{t('Search in Settings')}</Button>
```